### ISSUE TITLE
Component "Value distribution" is splitted after load workflow

### ISSUE THREAD

**Issue by KarelHouzvicka** [datacleaner/DataCleaner#412]
**Version: 4.0.4**
### Description
Create the workflow with Value distribution Analyzer. This Analyzer has two or more inputs fiels. Component has in workflow two or more input arrows.
Save it.
Close workflow and load it again.
Now is the analyzer on the same position but is splitted. Each analyzer has only one input with one arrow.

**Comment by kaspersorensen** [comment]
Thanks for this bug report. I have confirmed it and can also shed a bit of technicality light over it:

The ValueDistributionAnalyzer class defines this column propery:

```
    @Inject
    @Configured(value = PROPERTY_COLUMN, order = 1)
    @ColumnProperty(escalateToMultipleJobs = true)
    InputColumn<?> _column;
```

Specifically the attribute escalateToMultipleJobs is responsible here. It is a hint to the job builder API that it may build multiple analyzer jobs from one builder which holds a list of columns.

Probably the right thing to do is to ensure that at the time of saving this job builder we only save one <analyzer> element instead of multiple. That way we can reload the job and have a builder instantiated with the same state as before.

**Comment by arjansh** [comment]
This issue was fixed in the context of #1030.
